usabilla-react-native/src/index.js

```diff
@@ -18,10 +18,18 @@ function loadFeedbackForm(formId) {
     UsabillaBridge.loadFeedbackForm(formId)
 }
 
-function showLoadedForm(event) {
+function showLoadedForm() {
     UsabillaBridge.showLoadedFrom()
 }
 
+function sendEvent(event) {
+    UsabillaBridge.sendEvent(event)
+}
+
+function resetCampaigns() {
```

Comment:
I think that we should keep the same naming as in our Native SDKs and call this method `Usabilla.resetCampaignData`

Reply:
👍